package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "ssh-compute",
+  "version": "0.1.0",
+  "description": "Github Action: Connect to GCP instances via SSH",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build -m src/main.ts -o dist/main && ncc build -m src/post.ts -o dist/post",
+    "lint": "eslint src/ --ext .ts,.tsx",
+    "format": "prettier --write **/*.ts",
+    "test": "mocha -r ts-node/register -t 150s 'tests/unit/*.test.ts' --exit",
+    "e2e-tests": "mocha -r ts-node/register -t 150s 'tests/e2e.test.ts' --exit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google-github-actions/ssh-compute.git"
+  },
+  "keywords": [
+    "actions",
+    "google",
+    "ssh compute",
+    "compute",
+    "ssh"
+  ],
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/google-github-actions/ssh-compute/issues"
+  },
+  "homepage": "https://github.com/google-github-actions/ssh-compute#readme",
+  "dependencies": {
+    "@actions/core": "^1.5.0",
+    "@actions/exec": "^1.1.0",
+    "@actions/tool-cache": "^1.7.1",
+    "@google-github-actions/actions-utils": "^0.1.2",
+    "@google-github-actions/setup-cloud-sdk": "^0.3.1"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.21",
```

Comment:
Can you do an audit and make sure you're using these deps? For example, I don't use lodash being used.

Reply:
So, let's move tests to another PR.
I have deleted all dev deps here.